### ISSUE TITLE
BR0 - Documentation update / Required Fields

### ISSUE THREAD

**Issue by jszasz** [ccswbs/hjckrrh#71]
The spec says that the URL is optional but it currently is required on the content type. Also there is no mention of the "order" field on the spec (its a required field currently). Also, Alt text should be optional so that you can have an null alt tag for decorative images.

**Comment by abanuog** [comment]
@brharp do we want to make the URL field "not required" ? The alt text looks like it is optional already.

**Comment by brharp** [comment]
@abanuog if we make the URL "not required" then the slideshow template will have to be updated to update the slideshow title (without setting a link URL). I think some people have asked for this option, so we could consider it.

**Comment by ushenoyuog** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#71 (Documentation ONLY) BR0 - Spec update / Required Fields](https://trello.com/c/jhYiYvT9/240-71-documentation-only-br0-spec-update-required-fields)

**Comment by ushenoyuog** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#71 (Documentation ONLY) BR0 - Spec update / Required Fields](https://trello.com/c/jhYiYvT9/240-71-documentation-only-br0-spec-update-required-fields)

**Comment by ushenoyuog** [comment]
Making URL optional in banner has been requested by a few different clients , most recently by HR , CPES and COA. While this is being prioritised , we will update the documentation to reflect the system as it is currently. 

When the fix is done the documentation will need to be updated once again. 

![image](https://cloud.githubusercontent.com/assets/23462414/22442485/99595ade-e709-11e6-9dc8-0f3df63ff716.png)

**Comment by mmafe** [comment]
Issue has been updated to reflect documentation updates instead of original specification updates.

**Comment by ushenoyuog** [comment]
This issue has been closed. the documentation is updated. If the URL needs to be made opional, sepearte issue will be created.
